deploy-agent/tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist=py27,py37,py38,py39
+envlist=py36,py38,py39
```

Comment:
maybe add a comment on the reason for downgrading the py37 to py36

Reply:
I think the next py version update will be 3.8 but just keep 3.7 here just in case.